examples/myGraph.ts

```diff
@@ -16,4 +18,12 @@ Post1.link(Comment1);
 Comment1.link(Comment1Like1);
 Comment1.link(Comment1Like2);
 
-new Graph([LonelyUser, User]).toHtml("myUserGraph", { showVerb: true });
+new Graph([LonelyUser, User]).toHtml(
+  "myUserGraph",
+  { showVerb: true },
+  {
+    fontFamily: "Courier New",
```

Comment:
nice, maybe the 2 config arguments could become one, hopefully the `mermaid.initialize(MMD_OPTIONS);` doesn't complain of the extraneous ones